scripts/conditionalMitigator/conditionalMitigator.ps1

```diff
@@ -0,0 +1,409 @@
+param (
+    [Parameter(Mandatory=$true)]
```

Comment:
If makes sense, you can also add this under a flag

Stop capture which is already running: pktmon stop

Start Capture: pktmon start --trace -p Microsoft-Windows-Host-Network-Service -l 4 -m circular -s 1000

Repro:

Stop capture: pktmon stop

Convert capture (inside the same node): pktmon etl2txt pktmon.etl

Reply:
Do you mean using the script to start and stop pktmon?

Reply:
Yes, in circular mode with 1 GB as log size.

Reply:
Currently, not planning on collecting trace logs. Once we have better trace logging support in HNS at appropriate level, we can plan on collecting trace logs. Another thing is, enabling trace logs at daemonset level could be an issue. Need to understand the memory/disk cost of doing so.